kyuubi-common/src/main/scala/org/apache/kyuubi/service/TFrontendService.scala

```diff
@@ -46,7 +46,7 @@ abstract class TFrontendService(name: String)
   extends AbstractFrontendService(name) with TCLIService.Iface with Runnable with Logging {
   import TFrontendService._
   private val started = new AtomicBoolean(false)
-  private lazy val hadoopConf: Configuration = KyuubiHadoopUtils.newHadoopConf(conf)
+  private lazy val _hadoopConf: Configuration = KyuubiHadoopUtils.newHadoopConf(conf)
```

Comment:
why thrift server should not reload hadoop conf ?

Reply:
Only KyuubiTBinaryFrontendService will reload the hadoop conf, the engine will not.